modules/distribution/src/repository/resources/dbscripts/apimgt/oracle.sql

```diff
@@ -716,7 +715,7 @@ CREATE TABLE IDP_LOCAL_CLAIM (
             IDP_ID INTEGER,
             CLAIM_URI VARCHAR(255) NOT NULL,
             DEFAULT_VALUE VARCHAR(255),
-            IS_REQUESTED VARCHAR(128) DEFAULT '0',
+	        IS_REQUESTED VARCHAR(128) DEFAULT '0',
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04